### PR TITLE
Improve nginx ingress doc

### PR DESCRIPTION
**Pull request by scraly**

### CODE REVIEW

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/translator.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.translations = yaml.load(fd)
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/banner.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.banners = yaml.load(fd)
WARNING: Empty alt attribute for image img_2903.jpg in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/domains/glue_registry/guide.nl-nl.md
WARNING: Empty alt attribute for image img_2900.jpg in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/domains/glue_registry/guide.nl-nl.md
WARNING: Empty alt attribute for image img_2901.jpg in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/domains/glue_registry/guide.nl-nl.md
WARNING: Empty alt attribute for image img_2902.jpg in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/domains/glue_registry/guide.nl-nl.md
WARNING: Other images have empty alt attributes
WARNING: The {attach} link in account/api/api_right_delegation/guide.en-ca.md cannot relocate account/api/api_right_delegation/images/ak-as-ca.png because its output location was already overridden. Falling back to {filename} link behavior instead.
WARNING: The {attach} link in account/api/api_right_delegation/guide.en-ca.md cannot relocate account/api/api_right_delegation/images/sequence01.png because its output location was already overridden. Falling back to {filename} link behavior instead.
WARNING: The {attach} link in account/api/api_right_delegation/guide.en-ca.md cannot relocate account/api/api_right_delegation/images/validate-ck-ca.png because its output location was already overridden. Falling back to {filename} link behavior instead.
WARNING: The {attach} link in account/api/api_right_delegation/guide.en-ca.md cannot relocate account/api/api_right_delegation/images/sequence02.png because its output location was already overridden. Falling back to {filename} link behavior instead.
WARNING: More {attach} warnings silenced.
WARNING: Unable to find `images/3734.png`, skipping url replacement.
WARNING: Unable to find `images/3735.png`, skipping url replacement.
WARNING: Unable to find `images/3736.png`, skipping url replacement.
WARNING: Unable to find `images/3738.png`, skipping url replacement.
WARNING: Other resources were not found and their urls not replaced
CRITICAL: RuntimeError: File /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/output/gb/en/kubernetes/installing-nginx-ingress/index.html is set to be overridden twice
```

File: pages/platform/kubernetes-k8s/securing-nginx-ingress-cert-manager/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+---
+title: Securing Nginx Ingress on OVHcloud Managed Kubernetes with cert-manager
+slug: installing-nginx-ingress
```
Comment (OVH-Team-Guides) [inline pages/platform/kubernetes-k8s/securing-nginx-ingress-cert-manager/guide.en-gb.md]: Hello @scraly , this slug already exists for the Installing Nginx page (https://docs.ovh.com/gb/en/kubernetes/installing-nginx-ingress/)
Please use another slug.

Comment (scraly) [reply pages/platform/kubernetes-k8s/securing-nginx-ingress-cert-manager/guide.en-gb.md]: Hi, I removed the draft doc, sorry :)

**Comment by OVH-Team-Guides** [comment]
Thanks for your contribution @scraly

**Pull request merged**